js/content.js

```diff
@@ -215,74 +215,39 @@ function injectChatOverlay(tabUrl) {
     twitchVideoPlayer = playerQuery;
     twitchChatLines = chatQuery[0];
 
-    // create 2d canvas (and keep a reference)
-    myCanvas = document.createElement('canvas');
-    myCanvas.id = "MyTwitchChatOverlay";
-    myCanvas.width = twitchVideoPlayer.offsetWidth;
-    myCanvas.height = twitchVideoPlayer.offsetHeight;
-    myCanvas.style.position = "relative";
-    myCanvas.style.top = "0";
-    myCanvas.style.left = "0";
-    myCanvas.style["pointer-events"] = "none";
-    myCanvas.style.visibility = "visible";
-    myCanvas.style.opacity = gRolloutOpacity;
+    myTextLayer = document.createElement('div');
+    myTextLayer.id = "MyTwitchChatTextOverlay";
+    myTextLayer.width = '100%';
+    myTextLayer.height = '100%';
```

Comment:
Another big win with the new implementation is that we no longer have to track the width of the parent div, removing all handlers for screen resizing, which were a pain to maintain.